src/components/SamlConfiguration/index.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Collapsible, Icon } from '@edx/paragon';
+import SamlProviderConfigForm from './SamlProviderConfigForm';
+import SamlProviderDataForm from './SamlProviderDataForm';
+import { camelCaseObject, snakeCaseFormData } from '../../utils';
+import LmsApiService from '../../data/services/LmsApiService';
+import LoadingMessage from '../LoadingMessage';
+import ErrorPage from '../ErrorPage';
+
+class SamlConfiguration extends React.Component {
+  state = {
+    providerConfig: undefined,
+    providerData: undefined,
+    error: undefined,
+    loading: true,
+    deleteEnabled: false,
+  };
+
+  componentDidMount() {
+    LmsApiService.getProviderConfig(this.props.enterpriseId)
+      .then((response) => {
+        this.setState({
+          providerConfig: response.data.results[0],
+          loading: false,
+        });
+      })
+      .catch(error => this.setState({
+        error,
+      }));
+    LmsApiService.getProviderData(this.props.enterpriseId)
+      .then((response) => {
+        this.setState({
+          providerData: response.data.results[0],
```

Comment:
would double check if the setStat calls throughout may need to preserve existing state

Reply:
See previous comment.